index.d.ts

```diff
@@ -1,17 +1,17 @@
-export = Analytics;
+export default Analytics;
 /**
  * Represents a generic object in the APIs
  * Use for parameters like context, traits etc.
  */
-interface apiObject {
+export type apiObject = {
```

Comment:
Why are we exporting it as a type and not as an interface like `integrationOptions`?

Reply:
Also, let's export other interfaces too?

Reply:
I was under the wrong impression that the interface can’t be exported this way. Updating it.